src/site/labs/objects-collections/collections.rst

```diff
@@ -107,6 +70,8 @@ this lab.
 #. Write a ``toString`` for the class
 #. Test the ``CourseList`` class by creating an instance of it and using the methods
 
+        * Be sure to check all methods
+
 
 
 Kattis Problems
```

Comment:
some of these are duped with the objects specific lab now -- does kattis keep track of stuff they've already done?

Reply:
Yeah. 

I believe all labs have this exact list, with the exception of the first lab, which, for some reason, has a subset.